websites/K/Kahoot/dist/metadata.json

```diff
@@ -22,9 +26,10 @@
 	},
 	"url": [
 		"kahoot.it",
-		"play.kahoot.it"
+		"play.kahoot.it",
+		"create.kahoot.it"
 	],
-	"version": "2.0.10",
+	"version": "3.0.0",
```

Comment:
please explain why this is breaking

Reply:
@EncryptedDev told me that this was breaking.
https://github.com/PreMiD/Presences/pull/5975#discussion_r835824127

Some reasons this could be breaking is that it adds a new domain, and a lot of the original code has been rewritten.

Reply:
Adding a new domain is minor not breaking. Recoding is also not breaking unless you remove support for something. @EncryptedDev you should also explain comments like that a bit better

Reply:
I assumed a bunch of routes were removed, didn’t see they were replaced by the switch-case